### PR TITLE
Fix service charm validation.

### PR DESCRIPTION
**Pull request by frankban**
Do not raise an exception when the charm is not specified in the service section.

### CODE REVIEW

**Comment by jujugui** [comment]
Test FAILed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-bundlelib/11/
Test FAILed.

File: jujubundlelib/validate.py [hunk 1]
```diff
@@ -137,22 +136,28 @@ def validate_services(validator, machines_used={}):
...
+            try:
+                charm = references.Reference.from_string(charm_url)
+            except ValueError as e:
                 validator.add_error(
-                    'bundles not allowed for service {}: {}'.format(
-                        service_name, charm))
+                    'invalid charm specified for service {}: {}'.format(
+                        service_name, pyutils.exception_string(e)))
+                charm = None
+            else:
```
Comment (fabricematrat) [inline jujubundlelib/validate.py]: Where is the if for this else ?

**Comment by bac** [comment]
:+1:

**Comment by jujugui** [comment]
Test PASSed.
Refer to this link for build results (access rights to CI server needed): 
http://ci.jujugui.org:8080//job/juju-bundlelib/14/
Test PASSed.

**Comment by frankban** [comment]
Thanks for the review.
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://localhost:8080/job/juju-bundlelib-merge

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://localhost:8080/job/juju-bundlelib-merge

**Pull request merged**